src/codeFlowLogin.ts

```diff
@@ -231,7 +231,7 @@ function createTerminateServer(server: http.Server) {
 		});
 	});
 	return async () => {
-		const result = new Promise<void>(resolve => server.close(resolve));
+		const result = new Promise<void>(() => server.close());
```

Comment:
What's the reasoning behind this change? Not passing the `resolve` could change the timing, possibly causing a race condition